cdap-docs/hydrator-manual/source/developing-plugins/plugin-basics.rst

```diff
@@ -13,15 +13,17 @@ Plugin Types
 In Cask Hydrator, these plugin types are presently used:
```

Comment:
Should we explain the relationship between the types and the classifications shown in Hydrator?